packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -14,10 +14,9 @@ import CitizenshipForm from '../CitizenshipModal/set-citizenship-form.jsx';
 import PasswordSelectionModal from '../PasswordSelectionModal/password-selection-modal.jsx';
 import QuestionnaireModal from '../QuestionnaireModal';
 import ResidenceForm from '../SetResidenceModal/set-residence-form.jsx';
-
 import validateSignupFields from './validate-signup-fields.jsx';
-
 import 'Sass/app/modules/account-signup.scss';
+import { trackEvent } from 'Utils/Analytics/analytics.ts';
```

Comment:
```suggestion
import { trackEventWithCache } from 'Utils/Analytics/analytics.ts';
```
I think we better to distinguish it for the original trackEvent